### PR TITLE
v0.2.0 release

### PR DESCRIPTION
**Pull request by rebelinux**
## How Has This Been Tested?

- Tested in an enterprise Active Directory environment with Windows Server 2019 and a forest funtional level of Windows2016Forest.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.

### CODE REVIEW

File: AsBuiltReport.Microsoft.AD.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
     "Report": {
         "Name": "Microsoft AD As Built Report",
-        "Version": "1.0",
+        "Version": "0.2.0",
```
Comment (tpcarman) [inline AsBuiltReport.Microsoft.AD.json]: This relates to the version of the document/report which is generated, not the ABR module.

File: AsBuiltReport.Microsoft.AD.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
         "Status": "Released",
-        "ShowCoverPageImage": true,
+        "ShowCoverPageImage": false,
```
Comment (tpcarman) [inline AsBuiltReport.Microsoft.AD.json]: This should default to `true`, as users may use their own custom style scripts. The default style script should not include the image content if it is not supported.

File: README.md [hunk 3]
```diff
@@ -34,53 +34,79 @@ Microsoft AD As Built Report is a PowerShell module which works in conjunction w
...
 ### Linux & macOS
-* .NET Core is required for cover page image support on Linux and macOS operating systems.
-    * [Installing .NET Core for macOS](https://docs.microsoft.com/en-us/dotnet/core/install/macos)
-    * [Installing .NET Core for Linux](https://docs.microsoft.com/en-us/dotnet/core/install/linux)
+
+- .NET Core is required for cover page image support on Linux and macOS operating systems.
+  - [Installing .NET Core for macOS](https://docs.microsoft.com/en-us/dotnet/core/install/macos)
+  - [Installing .NET Core for Linux](https://docs.microsoft.com/en-us/dotnet/core/install/linux)
 
 ❗ If you are unable to install .NET Core, you must set `ShowCoverPageImage` to `False` in the report JSON configuration file.
```
Comment (tpcarman) [inline README.md]: May require an additional note to highlight that this will not currently work on Linus & MacOS

**Review by tpcarman — CHANGES_REQUESTED** [review]

File: AsBuiltReport.Microsoft.AD.json (hunk 1 above)
Comment (rebelinux) [reply AsBuiltReport.Microsoft.AD.json]: that's the drawback of using search&replace haha

File: AsBuiltReport.Microsoft.AD.json (hunk 2 above)
Comment (rebelinux) [reply AsBuiltReport.Microsoft.AD.json]: Done!

File: README.md (hunk 3 above)
Comment (rebelinux) [reply README.md]: !Done

**Comment by rebelinux** [comment]
All recommendations have now been addressed. Thanks!

**Review by tpcarman — APPROVED** [review]

**Pull request merged**